utilities/transactions/write_unprepared_txn_db.cc

```diff
@@ -373,8 +373,8 @@ Iterator* WriteUnpreparedTxnDB::NewIterator(const ReadOptions& options,
 
   // Currently, the Prev() iterator logic does not work well without snapshot
   // validation. The logic simply iterates through versions of a key in
-  // ascending seqno order, stopping a non-visible key and returning the last
-  // visible key.
+  // ascending seqno order, stopping at the first non-visible key and returning
```

Comment:
s/key/value or s/key/version
We are iterating over different values of the same key. Same comment for the next line.